docs/servers/bedrock.md

```diff
@@ -0,0 +1,321 @@
+
+The following identifiers are used for the available compression methods:
+- Zlib: 0x00
+- Snappy: 0x01
+- No Compression: 0xFF or 0xFFFF (in [NetworkSettings](#network-settings))
+
+This compression ID is stored before every gamepacket as am u8 and in the [NetworkSettings](#network-settings), where it is an u16 and defines the default compression method to use.
+
+## Encryption
+
+To be documented...
+
+## Caching
+
+To be documented...
+
+## Login Process
+
+The Bedrock Protocol's login sequence is made up of multiple stages, these being:
+- PreLogin
+- Login
+- Spawn
+- Play
+
+### Network Settings Request
+(Client -> Server)
+
+[Reference in the Official Docs](https://mojang.github.io/bedrock-protocol-docs/html/RequestNetworkSettingsPacket.html)
+
+Since v554 (v1.19.20), this is the first packet sent by the client.
+The NetworkSettingsRequestPacket has one single field that being the current protocol version of the client.
+
+This is the first PreLoin packet.
+
+### Network Settings
+(Server -> Client)
+
+[Reference in the Official Docs](https://mojang.github.io/bedrock-protocol-docs/html/NetworkSettingsPacket.html)
+
+Used to set up information for the connection, this is where the compression is set and initialized.
+Refer to [Compression](#compression) to find out more about compression here.
+
+
+This is the last PreLoin packet.
+
+### Login
+(Client -> Server)
+
+[Reference in the Official Docs](https://mojang.github.io/bedrock-protocol-docs/html/LoginPacket.html)
+
+The LoginPacket contains a lot of information on the client, such as the client protocol version (again).
+This should not be used since it is deprecated in newer versions.
+
+It also contains a JSON array of [JWTs](https://jwt.io/introduction) called the certificate chain which is encoded as a String.
+These JWTs contain verified information about the client. The array contains at minimum 1 JWT (Not authenticated to Xbox Live services), and at maximum 3 JWTs (Is authenticated to Xbox Live services).
+They store data such as the players display name, xuid and uuid. As well as sandbox- and title ID, when authenticated.
+If the 2nd and 3rd JWT are missing then the player has not signed in to their Xbox live account and is currently not authenticated, the sent data can hence why not be trusted.
+
+The LoginPacket also contains another JWT encoded as a String, that being the raw token.
+It contains information about the player such as:
+- SelfSignedId
+- ServerAddress = (unresolved url if applicable)
+- ClientRandomId
+- SkinId
+- SkinData
+- SkinImageWidth
+- SkinImageHeight
+- CapeData
+- CapeImageWidth
+- CapeImageHeight
+- SkinResourcePatch
+- SkinGeometryData
+- SkinGeometryDataEngineVersion
+- SkinAnimationData
+- PlayFabId
+- AnimatedImageData = Array of:
+  - Type
+  - Image
+  - ImageWidth
+  - ImageHeight
+  - Frames
+  - AnimationExpression
+- ArmSize
+- SkinColor
+- PersonaPieces = Array of:
+  - PackId
+  - PieceId
+  - IsDefault
+  - PieceType
+  - ProuctId
```

Comment:
Typo?
```suggestion
  - ProductId
```

Reply:
Thats a typo, but to my defense - It's a typo made by Mojang...